client/blocks/plugins-scheduled-updates/config.ts

```diff
@@ -1,3 +1,3 @@
-export const MAX_SCHEDULES = 2;
+export const MAX_SCHEDULES = 24;
```

Comment:
I'd be fine with setting no explicit limit and just have the "1 schedule/hour" rule take over.